<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Busbar</class>
 <widget class="QDialog" name="Busbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Busbar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelUid">
           <property name="text">
            <string>UID                  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Alias">
           <property name="text">
            <string>Alias</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditUid"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditName"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAlias"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelRatedVoltage">
         <property name="text">
          <string>Rated Voltage [kV]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxRatedVoltage"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelDescription">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditDescription"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
